### DISCUSSION TITLE
apache/dolphinscheduler-standalone-server:3.1.5 如何在workflow之间设置依赖

### DISCUSSION THREAD

**Discussion by yeangPan** [apache/dolphinscheduler#15608] [Q&A]
我有workflow1和workflow2，workflow1下面有任务test1和test2,workflow2下面有任务test3和test4,如何设置workflow2依赖workflow1。这两个workflow都上线了

**Comment by CHINAJAVABoy** [comment]
里面有dependent节点 或者子节点都可以实现依赖问题
